### ISSUE TITLE
Website is dead ?

### ISSUE THREAD

**Issue by Doimatcuabautroi** [vladignatyev/crx-extractor#13]
Hi guy, I love your project. My browser does not allow to install some extension with response "They are malware". I know your website and it can add bypass that browser. But website look like dead. Could you help me.
Website: https://crxextractor.com/
Dear.

**Comment by nonetrix** [comment]
Yes same for me 503 bad gateway, I tried self-hosting it no luck with that

Seem you need to build it somehow? I have no idea how to do any of this

edit:
```
git clone https://github.com/vladignatyev/crx-extractor
cd crx-extractor
npm run
npm start
```

**Comment by oleteacher** [comment]
I have been reluctant to share a working link because did not want to step on author's party. We have setup a unlisted version to use in classroom so students can experiment with extensions. We also added support for Edge.

https://qiksoft.com/tools/crx-extractor/

Not sure if this will be a permanent site, but use it freely for now:)

**Comment by Anxiemon** [comment]
> https://crxextractor.com/ Dear.

It isn't really dead, check it out!

**Comment by vladignatyev** [comment]
@oleteacher @Anxiemon @nonetrix @Doimatcuabautroi Thank you for your interest about CRXExtractor. Well done @oleteacher!
I had no plans on this project for years, but I see now that it can help people like you. Would you mind to share with me your opinions about missed features, use-cases and how CrxExtractor could help you? Please do it here or write me back to my e-mail ya.na.pochte@gmail.com
I'll be happy to contribute more and update the website and this tool itself to match your expectations and requests!

**Comment by nonetrix** [comment]
Perhaps you could merge the changes that @oleteacher has made or give him commit rights via a GitHub organization

**Comment by vladignatyev** [comment]
Will check out next week. Have a good weekend!

**Comment by nonetrix** [comment]
But they do seem to be promoting some fishy things, I am not entirely sure what these are but they seem to be monetary. I won't accuse them of being scams definitively, they just look extremely suspicious and spammy in my opinion 
![image](https://github.com/vladignatyev/crx-extractor/assets/45698918/9859080b-3c3e-47c8-8b74-f8da77e7711a)

![image](https://github.com/vladignatyev/crx-extractor/assets/45698918/7dc25969-8b47-4656-89d4-eaf8ec350371)
![image](https://github.com/vladignatyev/crx-extractor/assets/45698918/f6ed3bad-b983-4bcf-8d9b-903eecb7cb77)

**Comment by 2braincells2go** [comment]
@nonetrix I have to ask: how is promoting affiliate products fishy? I visited the site and see nothing "fishy". Looks like people trying to make a living to me.

**Comment by 2braincells2go** [comment]
@vladignatyev I know @oleteacher personally, have worked with her in education field for over 20 years. I will alert her if you like about providing you with her changes to library. Pretty sure she would be more than happy to provide any code you may need if not already in her GitHub repository.
